src/rgw/rgw_amqp.cc

```diff
@@ -865,6 +897,10 @@ class Manager {
       return nullptr;
     }
 
+    // when the url starts with amqps, amqp_parse_url will assume SSL, and we should keep it that way
```

Comment:
don't think we need an explicit "use-ssl" argument here. better to just follow the schema of the url and decide based on "amqp" vs "amqps"